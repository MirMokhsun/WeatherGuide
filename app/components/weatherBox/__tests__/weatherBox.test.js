import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import WeatherBox from '..';

describe('WeatherBox', () => {
    test('WeatherBox snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<WeatherBox />);
        expect(snapshot).toMatchSnapshot();
    });
});