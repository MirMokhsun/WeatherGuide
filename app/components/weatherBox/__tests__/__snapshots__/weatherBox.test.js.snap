// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherBox WeatherBox snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgba(0,0,0,0.3)",
      "borderRadius": 3,
      "height": 160,
      "marginLeft": 5,
      "marginTop": 15,
      "width": 740,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "rgba(33, 57, 156,0.3)",
        "borderRadius": 3,
        "flexDirection": "row",
        "height": 100,
        "justifyContent": "space-between",
        "marginTop": 5,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "column",
          "justifyContent": "center",
        }
      }
    >
      <Text
        style={
          Object {
            "fontFamily": "Roboto-Black",
            "fontSize": 18,
          }
        }
      >
         fr 2/28/2020
      </Text>
      <Text
        style={
          Object {
            "fontFamily": "Roboto-Black",
            "fontSize": 18,
          }
        }
      >
        4:12
      </Text>
    </View>
    <View
      style={
        Object {
          "flex": 1.25,
          "justifyContent": "center",
        }
      }
    >
      <Text
        style={
          Object {
            "fontFamily": "RobotoThinItalic-pqYK",
            "fontSize": 90,
          }
        }
      >
        7°c
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "column",
          "justifyContent": "center",
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../app/assets/logos/sun.png",
          }
        }
        style={
          Object {
            "maxHeight": 65,
            "maxWidth": 65,
          }
        }
      />
      <Text
        style={
          Object {
            "fontFamily": "Roboto-Black",
            "fontSize": 18,
          }
        }
      >
        Sunny
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(33, 57, 156,0.3)",
        "borderRadius": 3,
        "flexDirection": "column",
        "justifyContent": "center",
        "marginTop": 2,
      }
    }
  >
    <Text
      style={
        Object {
          "fontFamily": "Roboto",
          "fontSize": 18,
        }
      }
    >
      Max: 10° Min: 5°
    </Text>
    <Text
      style={
        Object {
          "fontFamily": "Roboto",
          "fontSize": 18,
        }
      }
    >
      Wind: East East-South 29km/h
    </Text>
  </View>
</View>
`;
